src/mame/jpm/jpmsru.cpp

```diff
@@ -384,10 +396,7 @@ void jpmsru_state::outputs_lc(address_map &map)
 	map(0x3c, 0x3d).w(FUNC(jpmsru_state::out_payout_2x50p_a_w));
 	map(0x3e, 0x3f).w(FUNC(jpmsru_state::out_payout_2x50p_b_w));
 	map(0x40, 0x4b).w(FUNC(jpmsru_state::out_disp_w));
-	/* Manual calls this "Loudspeaker"
-	   It's toggled at a regular speed, not fast like a DAC, so looks like a
-	   fixed beeper of some kind. No other information on what this is. */
-	map(0x4e, 0x4f).nopw();
+	map(0x4e, 0x4f).w("dac", FUNC(dac_bit_interface::data_w)).umask16(0xff00);
 	map(0x50, 0x61).w(FUNC(jpmsru_state::out_logicext_w));
 	map(0x62, 0x63).nopw(); // 2x50p B payout 2
 	map(0x6c, 0x6d).w(FUNC(jpmsru_state::out_10p_lockout_w));
```

Comment:
This memory map only makes sense for systems that have the DAC – it will blow up if the DAC is not present.  It should be moved to the class with the DAC.  You get the added benefit that you can use `m_dac` in the memory map rather than repeating the literal tag.

Reply:
Fixed.